ClosedXML_Tests/Excel/DataValidations/DataValidationTests.cs

```diff
@@ -27,11 +27,11 @@ public void Validation_1()
 
             ws.Cell("A1").SetValue("Cell below has Validation Only.");
             cell = ws.Cell("A2");
-            cell.DataValidation.List(ws.Range("$E$1:$E$4"));
+            cell.SetDataValidation().List(ws.Range("$E$1:$E$4"));
```

Comment:
Why did you change `.DataValidation.` to `.SetDataValidation().` in the tests?

Reply:
Is there no way to avoid this breaking change? This breaking change is very subtle and will confuse a lot of users. Please reply here in this thread, not the main thread.

Reply:
>Otherwise there would be NullReferenceException. I should have mentioned that this PR contains breaking changes

You're right, better to avoid breaking changes. Will change

Reply:
You still have to change the settings back? Or maybe test both?

Reply:
Uh, sorry, missed there were multiple changes in this test.